src/ayab/com.cpp

```diff
@@ -101,6 +101,10 @@ void Com::send_indState(Carriage_t carriage, uint8_t position,
  */
 void Com::onPacketReceived(const uint8_t *buffer, size_t size) {
   switch (buffer[0]) {
+  case reqInit_msgid:
```

Comment:
Why not add this parameter to reqStart ? What about adding a Shield HW version as well, e.g. ignore right Hall sensor with current HW/handle with next one ?

Reply:
Indeed, you need machineType if the capability to run a machine-dependent carriage detection is required ... which is the case for the right hall sensor already/OK.
Wrt cnfinfo, I see FW version but no HW version (yet ?)

Reply:
I don't think we have a way of detecting HW version from the firmware yet.